src/fiber/fiber.mli

```diff
@@ -227,6 +227,29 @@ module Mvar : sig
   val write : 'a t -> 'a -> unit fiber
 end
 
+module Svar : sig
```

Comment:
Let's expand the comment a bit.

```suggestion
(** A "state variable" [Svar.t] differs from [Ivar.t] in that it can be updated
    multiple times. It is particularly useful if you'd like to observe the state
    of a process by subscribing to "interesting" (from your perspective) events.
    
    For example, a build system can have a complex state machine, switching 
    between states like "waiting for source file changes", "building", etc., by
    calling [Svar.write] on the corresponding state variable. If you don't care
    about most of the states but would like to observe all errors, you can use
    [Svar.wait ~until:is_error] to be woken up when an error happens. Other
    observers may have different preferences, expressed with different [until]
    predicates. You can also use the non-blocking [Svar.read] if you'd like to
    use polling instead. *)
module Svar : sig
```